### DISCUSSION TITLE
Zasoby do tworzenia kuponów

### DISCUSSION THREAD

**Discussion by hahyes** [allegro/allegro-api#4764] [Suggestions - rejected]
Witam,

czy planujecie dodanie zasobu/zasobów do tworzenia kampanii kuponowych? Sugeruję to jako nową funkcjonalność w API.

Pozdrawiam.

**Comment by Lukasz-Zurek** [comment]
Nie planujemy wdrożenia tego typu funkcjonalności. Dziękuję jednak za sugestię. Poinformujemy, jeśli coś się zmieni w tym temacie.
